### DISCUSSION TITLE
"You have X QSOs today" - should it use local time?

### DISCUSSION THREAD

**Discussion by violarulan** [wavelog/wavelog#1140] [General]
https://github.com/wavelog/wavelog/blob/b313e0aed80c5278bca0fe5a513ebc1cce98e589/application/models/Logbook_model.php#L2617

We use UTC time in some places to limit the time range of 'a day'. In the index page, a tip is displayed to show that X QSOs are made within a day.

Regarding the readability, is it necessary to bring timezones to it? It can avoid some confusion from some users.

We may need to discuss that.

**Comment by xyz667** [comment]
I would say that the current "day" for most people  is what they can read on their (local) calendar and clock.
In western Europe where the difference to UTC is 1 or 2 hours  it could be neglected maybe, but in other parts of the world I would be confused if a significant part of yesterday's QSOs would be counted on the current day.
So yes, local time preferable IMO.
